layouts/ucf-post-list-card.php

```diff
@@ -60,6 +65,9 @@ function ucf_post_list_display_card( $content, $posts, $atts ) {
 						<?php endif; ?>
 						<div class="ucf-post-list-card-block">
 							<h3 class="ucf-post-list-card-title"><?php echo $item->post_title; ?></h3>
+							<?php if( $atts['show_excerpt'] ) : ?>
+								<p class="ucf-post-list-excerpt-text ucf-post-list-card-text"><?php echo $item_excerpt; ?></p>
```

Comment:
Please use a `<div>` to wrap the excerpt here instead of a `<p>` tag--excerpts by default can be wrapped in `<p>` tags; we just override this behavior in the UCF WordPress Theme.  Using a `<div>` instead will be more compatible with other themes.

Reply:
I'm putting together a revision and will include that. However I've noticed the date field is in `<p>` tags: 
`<p class="ucf-post-list-card-text"><?php echo $date; ?></p>`

Would you like me to change that to `<div>` in my revision?

Reply:
The date being wrapped in a `<p>` is fine as-is, since `$date` should always be a date string with no additional formatting--but thanks for looking out 🙂